cosmo_tester/test_suites/cluster/cfy_cluster_manager_test.py

```diff
@@ -79,24 +81,11 @@ def test_create_three_nodes_cluster(three_vms, three_nodes_config_dict,
 def test_create_nine_nodes_cluster(nine_vms, nine_nodes_config_dict,
                                    test_config, ssh_key, logger):
     """Tests that a nine nodes cluster is successfully created."""
-    node1, node2, node3, node4, node5, node6, node7, node8, node9 = nine_vms
-    for i, node in enumerate([node1, node2, node3, node4, node5, node6,
-                              node7, node8, node9]):
-        node_num = (i % 3) + 1
-        if i < 3:
-            node_name = 'rabbitmq-{0}'.format(node_num)
-        elif i < 6:
-            node_name = 'postgresql-{0}'.format(node_num)
-        else:
-            node_name = 'manager-{0}'.format(node_num)
-
-        nine_nodes_config_dict['existing_vms'][node_name].update({
-            'private_ip': str(node.private_ip_address),
-            'public_ip': str(node.ip_address)
-        })
+    nodes_list = [node for node in nine_vms]
```

Comment:
What is the type of the `nine_vms` ? is not a list ? why do we convert it again ?

Reply:
`nine_vms` is a generator https://github.com/cloudify-cosmo/cloudify-system-tests/blob/8d5428cb31304fda7837725340449dfcb6ca2964/cosmo_tester/test_suites/cluster/conftest.py#L133, this is why we convert it into a list.